modules/consentManagement.js

```diff
@@ -194,7 +194,7 @@ export function requestBidsHook(fn, reqBidsConfigObj) {
   // preserves all module related variables for the current auction instance (used primiarily for concurrent auctions)
   const hookConfig = {
     context: this,
-    args: arguments,
+    args: [reqBidsConfigObj],
```

Comment:
@jsnellbaker does this fix the issue you referred to?

Reply:
That does appear to also fix the issue.  Thanks.